source/connections-survive-step-down/tests/README.rst

```diff
@@ -71,8 +71,19 @@ Perform the following operations:
 - If the driver implements the `CMAP`_ specification, verify that no new `PoolClearedEvent`_ has been
   published. Otherwise verify that `connections.totalCreated`_ in `serverStatus`_ has not changed.
 
+**Note:** The "replSetStepDown" command often fails with the following
+transient error (see `SERVER-48154`_)::
 
+  {
+    "ok" : 0,
+    "errmsg" : "Unable to acquire X lock on '{4611686018427387905: ReplicationStateTransition, 1}' within 1000ms. opId: 922, op: conn30, connId: 30.",
```

Comment:
Not a big deal, but I think the error message here would be "within 5000ms", though I've observed slight discrepencies:

```
test> db.adminCommand({replSetStepDown: 5, force: true})
{
	"ok" : 0,
	"errmsg" : "Unable to acquire X lock on '{4611686018427387905: ReplicationStateTransition, 1}' within 4999ms. opId: 2588184, op: conn1288, connId: 1288.",
	"code" : 24,
	"codeName" : "LockTimeout",
	"$clusterTime" : {
		"clusterTime" : Timestamp(1592331047, 1),
		"signature" : {
			"hash" : BinData(0,"AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
			"keyId" : NumberLong(0)
		}
	},
	"operationTime" : Timestamp(1592331047, 1)
}
```

Reply:
The error is just an example failure, I don't think the actually time in the error message matters.